### PR TITLE
Cloud templates

### PR DESCRIPTION
**Pull request by kelseiv**
Closes https://github.com/influxdata/docs-v2/issues/1551

cc'd several folks on review here as a heads up. Please review if you can.

Note, this doesn't include OSS doc updates. Staying tuned from @russorat to be sure that's a go for OSS 2.0 RC

### CODE REVIEW

File: content/influxdb/v2.0/influxdb-templates/cloud.md [hunk 1]
```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
...
+
+    - Buckets. [Update buckets](/influxdb/v2.0/organizations/buckets/update-bucket/)
+    - Dashboards. [Control dashboards](/influxdb/v2.0/visualize-data/dashboards/control-dashboard/)
+    - Labels. [Edit labels](/influxdb/v2.0/visualize-data/labels/#edit-a-label)
+    - Checks. [Update checks](/influxdb/v2.0/monitor-alert/checks/update/)
+    - Notification endpoints. [Update notification endpoints](/influxdb/v2.0/monitor-alert/notification-endpoints/update/)
+    - Notification rules. [Update notification rules](/influxdb/v2.0/monitor-alert/notification-rules/update/)
+    - Telegraf configurations. [Update a Telegraf configurations](/influxdb/v2.0/telegraf-configs/update/)
+
+7. (Optional) Add or delete resources from the template, and then use the `influx` CLI to [export resources to a new template (JSON or YAML file)](/influxdb/v2.0/influxdb-templates/create/#export-all-resources).
```
Comment (kelseiv) [inline content/influxdb/v2.0/influxdb-templates/cloud.md:49]: Figured links to add/delete resources docs were extra, so left them out. Will include if we feel it's better to be explicit here.

File: content/influxdb/v2.0/influxdb-templates/cloud.md [hunk 2]
```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
...
-to apply and manage templates in your InfluxDB Cloud account.
+To use templates in InfluxDB Cloud, you have a couple options:
+
+- [Install and customize templates directly in the Cloud UI](#install-and-customize-a-template-in-the-cloud-ui)
+- [Apply and manage templates using the `influx` CLI](#apply-and-manage-templates-using-the-influx-cli)
+
+## Install and customize a template in the Cloud UI
+
+{{% note %}}
+To ensure the template works, complete the **ReadMe** instructions after installing a template. For example, you may need to create a bucket with a specific name in your Cloud instance before the template can be used.
```
Comment (hoorayimhelping) [inline content/influxdb/v2.0/influxdb-templates/cloud.md:25]:
```suggestion
To ensure the template works, complete the **ReadMe** instructions after installing a template. For example, you may need to configure telegraf to write specific data to your Cloud instance before the template can be used.
```

suggestion: telegraf is by far the most common configuration instruction we ran into when going over the testing document: https://docs.google.com/spreadsheets/d/1-jLiZtxOImQBKLZf5uWAptxfJn_LciBgFmzvChbB1oo/edit#gid=0

File: content/influxdb/v2.0/influxdb-templates/cloud.md [hunk 3]
```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
...
+{{% /note %}}
+
+1. Under **Settings** icon in the navigation bar.
+
+    {{< nav-icon "settings" >}}
 
+2. Click **Templates**.
+3. (Optional) If you already have a template URL, you can skip this step. Click **Browse Community Templates** to explore templates and obtain a template URL (YAML or JSON file).
+4. Enter the URL to the template's YAML or JSON file (from the [InfluxData Community Templates](https://github.com/influxdata/community-templates) repository), and then click **Lookup Template**.
+The Template Installer displays the template name and the number of resources included in the template (for example, dashboards, Telegraf configurations, buckets, and more).
```
Comment (hoorayimhelping) [inline content/influxdb/v2.0/influxdb-templates/cloud.md:35]: might want to mention that some resources can be unchecked, meaning they won't be applied

File: content/influxdb/v2.0/influxdb-templates/cloud.md [hunk 4]
```diff
@@ -15,9 +14,48 @@ influxdb/v2.0/tags: [templates]
 products: [cloud]
 ---
 
-Use the [`influx` command line interface (CLI)](/influxdb/v2.0/reference/cli/influx/)
-to apply and manage templates in your InfluxDB Cloud account.
+To use templates in InfluxDB Cloud, you have a couple options:
+
+- [Install and customize templates directly in the Cloud UI](#install-and-customize-a-template-in-the-cloud-ui)
```
Comment (hoorayimhelping) [inline content/influxdb/v2.0/influxdb-templates/cloud.md:19]: we use the term Apply in the cli and Install in the UI. I think it's out of scope for this PR, but we should probably unify the terms to one or the other - it's a bit confusing that there are two words that mean the same thing.

**Review by hoorayimhelping — COMMENTED** [review]
This is a doc some of used to test templates. It has a few notes about the UX of installing templates. One of the key takeaways for me was that almost every template had some telegraf component to it. Might be helpful (or it might be a total waste of your time 🤷‍♂️)

https://docs.google.com/spreadsheets/d/1-jLiZtxOImQBKLZf5uWAptxfJn_LciBgFmzvChbB1oo/edit#gid=0

File: content/influxdb/v2.0/influxdb-templates/cloud.md (hunk 4 above)
Comment (kelseiv) [reply content/influxdb/v2.0/influxdb-templates/cloud.md:19]: I'll update globally once we update the corresponding terms in the UI

**Pull request merged**